crates/db_schema/src/impls/post.rs

```diff
@@ -96,6 +99,21 @@ impl Post {
       .await
   }
 
+  pub async fn list_for_sitemap(
+    pool: &mut DbPool<'_>,
+  ) -> Result<Vec<(DbUrl, chrono::NaiveDateTime)>, Error> {
+    let conn = &mut get_conn(pool).await?;
+    post
+      .select((ap_id, coalesce(updated, published)))
+      .filter(local)
```

Comment:
I've never seen this before, I hope its doing `local.eq(true)`

Reply:
Too much Javascript at work, sorry :sweat_smile: It does compile and it doesn't panic at runtime, but I also haven't been able to test this fully end-2-end since I don't have any posts in my local db. Tried to look at the diesel source to figure out what is expected here, but didn't have much success.

@dessalines Should I make a PR to change this to `.filter(local.eq(true)`, since we know it works?